File: traductor/ejemplos/test3_colecciones.cs
using System;
using System.Collections.Generic;

public class Contenedor
{
    private List<string> items;

    public void agregar(string item)
    {
        items.Add(item);
    }

    public void listar()
    {
        foreach (string s in items)
        {
            Console.WriteLine(s);
        }
    }
}

File: traductor/ejemplos/test3_colecciones.java
import java.util.*;

public class Contenedor {
    private List<String> items;

    public void agregar(String item) {
        items.add(item);
    }

    public void listar() {
        for (String s : items) {
            System.out.println(s);
        }
    }
}
